e2e/tests/workspace/workspace-delete.spec.ts

```diff
@@ -0,0 +1,70 @@
+import WorkspacesPage from 'app/page/workspaces-page';
+import {findWorkspace, signIn} from 'utils/test-utils';
+import WorkspaceCard from 'app/component/workspace-card';
+import {EllipsisMenuAction, WorkspaceAccessLevel, LinkText} from 'app/text-labels';
+import Modal from 'app/component/modal';
+
+
+describe('Delete workspace', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+  /**
+   * Test:
+   * - Find an existing workspace. Create a new workspace if none exists.
+   * - Select "Delete" thru the Ellipsis menu located on the Workspace card.
+   * - Get the content on the modal
+   * - Click the delete workspace button on the modal
+   * - Navigate to the "View Workspaces" page
+   * - Verify that the workspace has been delete
+   */
+  
+  describe('From "Your Workspaces" page using Workspace card ellipsis menu', () => {
```

Comment:
I think it's the `Home` page where delete happens, right?

Reply:
the delete happens on"YOUR WORKSPACES" page.